nucypher/cli/commands/ursula.py

```diff
@@ -442,3 +449,143 @@ def _pre_launch_warnings(emitter, dev, force):
         emitter.echo(DEVELOPMENT_MODE_WARNING, color='yellow', verbosity=1)
     if force:
         emitter.echo(FORCE_MODE_WARNING, color='yellow', verbosity=1)
+
+
+BACKUP_ENCRYPTION_SETTINGS = {'encryption': pyzipper.WZ_AES, 'nbits': 256}
+WORKER_ARCHIVE_ROOT = "worker"
+KEYSTORE_ARCHIVE_ROOT = "keystore"
+
+
+class BackupCliOptions:
+    __option_name__ = 'backup_options'
+
+    def __init__(self,
+                 worker_path: str,
+                 keystore_path: str,
+                 backup_path: str,
+                 password: str,
+                 overwrite: bool,
+                 ) -> None:
+        self.worker_path = Path(worker_path)
+        self.keystore_path = Path(keystore_path)
+        self.backup_path = Path(backup_path)
+        self.password = password.encode()
+        self.overwrite = overwrite
+
+
+backup_cli_options = group_options(
+    BackupCliOptions,
+    password=click.option('--password',
+                          help="Enter backup password",
+                          prompt="Enter backup password",
+                          hide_input=True,
+                          confirmation_prompt=True),
+    worker_path=click.option('--worker-path',
+                             help="Path to Ursula worker directory",
+                             prompt="Path to Ursula worker directory",
+                             type=click.Path(),
+                             required=True,
+                             default=DEFAULT_CONFIG_ROOT),
+    keystore_path=click.option('--keystore-path',
+                               help="Path to Ursula keystore directory",
+                               prompt="Path to Ursula keystore directory",
+                               type=click.Path(),
+                               required=True,
+                               default=Path.home() / ".ethereum/keystore"),  # TODO: use env var?
```

Comment:
This only works in Linux. OS X uses a different path (e.g., `/Users/foo/Library/Ethereum/keystore`)